### ISSUE TITLE
Empty Excel file , after changing the view name

### ISSUE THREAD

**Issue by NouhaGHA** [OpenNTF/POI4Xpages#47]
Hi, 
I'm working with POI for a while  , i've changed the view name used as a resource, and the Excel report now is generated blank with empty clns, even when i put back the old configurations from a backup data base, i still have  the same problem. i've verified the clns and the view names 20 times, is there any thing that i should do ? 
Thanks

**Comment by guedeWebGate** [comment]
Can you pleases share you XPages Code / the version of POI4XPages. Btw. does the view contains data to export for the given user?

**Comment by NouhaGHA** [comment]
Of course , im working Domino 9.0.1 FP8
POI 1.2.6.201312211419
Of course the view contains data , 
Actualy, my code was working fine , when i changed the view's name , nothing is working again 

<?xml version="1.0" encoding="UTF-8"?>
<xp:view xmlns:xp="http://www.ibm.com/xsp/core"
	xmlns:wgpoi="http://www.webgate.biz/xsp/poi">
	<xp:br></xp:br>
	<wgpoi:spreadsheet id="shContacts"
		downloadFileName="listeCongés.xlsx">
		<wgpoi:this.templateSource>
			<wgpoi:resourcetemplate fileName="Book1.xlsx">
			</wgpoi:resourcetemplate>
		</wgpoi:this.templateSource>
		<wgpoi:this.spreadsheets>
			<wgpoi:table name="Congés" create="false">

				<wgpoi:this.exportDefinitions>
					<wgpoi:data2rowExport stepSize="1" var="row"
						index="index" startRow="2">
						<wgpoi:this.dataSource>
							<wgpoi:dominoViewSource
								viewName="EtatContact">
							</wgpoi:dominoViewSource>
						</wgpoi:this.dataSource>
						<wgpoi:this.columns>
							<wgpoi:columnDefinition columnNumber="0"
								columnTitle="Name" rowShift="0">



								<wgpoi:this.cellStyle>
									<wgpoi:cellStyle
										alignment="ALIGN_LEFT">
									</wgpoi:cellStyle>
								</wgpoi:this.cellStyle>
							</wgpoi:columnDefinition>
							<wgpoi:columnDefinition columnNumber="1"
								columnTitle="startDate" rowShift="0">
							</wgpoi:columnDefinition>
							<wgpoi:columnDefinition columnNumber="2"
								columnTitle="EndDate" rowShift="0">
							</wgpoi:columnDefinition>
							<wgpoi:columnDefinition columnNumber="3"
								columnTitle="Period" rowShift="0">
							</wgpoi:columnDefinition>
							<wgpoi:columnDefinition columnNumber="4"
								columnTitle="Motif" rowShift="0">
							</wgpoi:columnDefinition>


						</wgpoi:this.columns>
					</wgpoi:data2rowExport>
				</wgpoi:this.exportDefinitions>
			</wgpoi:table>
		</wgpoi:this.spreadsheets>
	</wgpoi:spreadsheet>
	<xp:br></xp:br>
	<xp:button value="Générer Rapport des congés" id="button1"><xp:eventHandler event="onclick" submit="true" refreshMode="complete">
	<xp:this.action>
		<wgpoi:generateWorkbook workbookId="shContacts"></wgpoi:generateWorkbook>
	</xp:this.action></xp:eventHandler></xp:button></xp:view>


Regards,

**Comment by guedeWebGate** [comment]
Was your code changed bevor you did go to FP8? I had the same issue, with 1.2.6 and FP8. The Designer was no longer capable to compile the XPages correctly. Can try it with POI4XPages 1.3.1. You have to update Designer and Server

**Comment by NouhaGHA** [comment]
Non, i didn't change the domino version, it is the same from the beginning , I appreciate your help i will download it and keep you in touch

**Comment by NouhaGHA** [comment]
Hi, sorry for taking so long to give you my feedback, i've downloaded the 1.3.1 version from the following link : 
 https://www.openntf.org/main.nsf/project.xsp?r=project/POI%204%20XPages/releases/E80C4FC9FB07E1E4852580E3006E02C7

But the library was empty , when i integrated the library to the updatesite base,it was empty , any idea ?
